src/BeautifulComments/BCBeautifulCommentsSmokeTest.class.st

```diff
@@ -13,3 +13,30 @@ BCBeautifulCommentsSmokeTest >> testRenderDoesNotRaiseError [
 							of: ClyRichTextClassCommentEditorToolMorph ]
 		raise: Error
 ]
+
+{ #category : #tests }
+BCBeautifulCommentsSmokeTest >> testSmokeSignal [
+	BeautifulComments captureErrors: true.
+	Smalltalk globals allClasses do: [ :cl |
```

Comment:
I prefer self class environment globals

Reply:
I will fix that - I just never can figure out how to do it.